app/services/creators/group_creator.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+#
+# Service object to create Groups.
+class GroupCreator < Creator
+  # Initialize a new GroupCreator
+  #
+  # @param [ActionController::Parameters] params The params object from
+  #   the GroupsController.
+  def initialize(params)
+    super(klass: Group, name_method: :name, params: params)
+    process_params
+  end
+
+  private
+
+  def process_params
+    @params = params.to_h.transform_keys(&:to_sym)
+    add_draw_to_params if params[:leader_id]
+    remove_blank_members if params[:member_ids]
+  end
+
+  def add_draw_to_params
+    @params.merge!(draw_id: User.find(params[:leader_id]).draw.id)
+  end
+
+  def remove_blank_members
+    @params[:member_ids] = params[:member_ids].reject(&:empty?)
+  end
+
+  def success(group)
+    { object: [group.draw, group], group: group,
```

Comment:
I was confused about this value for `object` but now I see you're using it to redirect to a nested resource, nice!